lib/pipeline/labels.go

```diff
@@ -39,3 +41,11 @@ func (p pipelineLabels) WithPipelineType(pipelineType string) pipelineLabels {
 	p["kratix-pipeline-type"] = pipelineType
 	return p
 }
+
+func (p pipelineLabels) WithRequestSHA(requestSHA []string) pipelineLabels {
+	if len(requestSHA) == 0 {
+		return p
+	}
+	p[KratixResourceHashLabel] = requestSHA[0]
+	return p
+}
```

Comment:
why is it a list of sha 🤔

Reply:
this function is called by `ConfigurePipelineLabels` , which sometimes is called without a request sha. i could change it to not be a slice, but would have to add the `if len...` on that function (which is maybe better?)

Reply:
ahh I see.

> but would have to add the if len... on that function (which is maybe better?)

 I think its worth changing, its bit more verbose for a clearer func def